modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1413,6 +1415,26 @@ private byte[] _decode(String fileBase64) {
 		}
 	}
 
+	private void _deleteObjectEntry(
+			DTOConverterContext dtoConverterContext,
+			com.liferay.object.model.ObjectEntry objectEntry)
```

Comment:
Can you rename this variable to `serviceBuilderObjectEntry`? we have other examples like this in this same class.